Student-Unity/src/main/java/csed/swe/studentunity/DAO/VerificationCodeRepo.java

```diff
@@ -0,0 +1,10 @@
+package csed.swe.studentunity.DAO;
+
+import csed.swe.studentunity.model.VerificationCode;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.Optional;
+
+public interface VerificationCodeRepo extends JpaRepository<VerificationCode, Long> {
+    Optional<VerificationCode> findVerificationCodeByEmail(String email);
```

Comment:
**ById**

Reply:
If you notice this table involves the verification codes of forget password and code of registered unverified users
and there are separate tables for user and unverified user so i can't use id of user and unverified user here as it may be equal each other as the id is generated sequentially so the only thing to uniquely identify the row in this table is by the email

Reply:
ok no problem